amlb/results.py

```diff
@@ -510,24 +556,34 @@ def __init__(self, predictions_df, info=None):
         self.type = DatasetType.regression
 
     def mae(self):
+        """Mean Absolute Error"""
         return float(mean_absolute_error(self.truth, self.predictions))
 
     def mse(self):
+        """Mean Squared Error"""
         return float(mean_squared_error(self.truth, self.predictions))
 
     def msle(self):
+        """Mean Squared Logarithmic Error"""
         return float(mean_squared_log_error(self.truth, self.predictions))
 
     def rmse(self):
+        """Root Mean Square Error"""
         return math.sqrt(self.mse())
 
     def rmsle(self):
+        """Root Mean Square Logarithmic Error"""
         return math.sqrt(self.msle())
 
     def r2(self):
+        """R^2"""
         return float(r2_score(self.truth, self.predictions))
 
 
+def higher_is_better(metric):
```

Comment:
This seems a bit hacky. Better to have either a dictionary mapping or metrics as classes ([example in AutoGluon](https://github.com/awslabs/autogluon/blob/master/core/src/autogluon/core/metrics/__init__.py#L17)).

Reply:
I can't disagree with you: it IS a bit hacky.
Ideally, there should be a class for each metric. It's probably something I'll do at some point to support custom metrics or other customizations in a more satisfying way that what was done in https://github.com/openml/automlbenchmark/pull/141.
If there's a demand for it, I'll do it.